README.md

```diff
@@ -100,38 +100,38 @@ To check that they're all correct, you can run the below command:
 ./gradlew checkLicenses
 ```
 
-This is run during the CI process, and will fail if the license is missing or not up to date, so make sure that you add
+This is run during the CI process and will fail if the license is missing or not up to date, so make sure that you add
 this to all new files you create.
 
 ## Create Custom Themes
 
-ATLauncher supports custom themes. The process is fairly straight forward, but may require a lot of trial and error.
+ATLauncher supports custom themes. The process is fairly straight forward but may require a lot of trial and error.
 
 First you must create a `MyThemeName.java` in the `src/main/java/com/atlauncher/themes/` directory. Your theme should
 extend one of the base ATLauncher themes depending on what you need:
 
 -   `Dark` is the default theme and is a dark theme. It's a good place to start with some defaults for new dark themes.
 -   `Light` is a light theme. It's a good place to start with some defaults for new light themes.
 -   `ATLauncherLaf` is a base class which every theme MUST at some point extend. It provides some defaults including our
-    brand colours and some defaults. This shouldn't be extended from unless you need absolute power.
```

Comment:
This is not incorrect, launchers base language including all documentation is en-gb not en-us